docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -321,31 +332,39 @@ You can modify the flex item placement through the following properties:
 
     **Figure: Flex**
 
-    ![Flex](media/flex.jpg)
+    ![Flex](media/flex.png)
 
     To create a layout with the proportions shown in the above figure:
 
-    ```
+    ```csharp
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
 
-    /// Place items horizontally
-    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
+    /// Resize flex container to window size
+    flexContainer.WidthResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.HeightResizePolicy = ResizePolicyType.FillToParent;
```

Comment:
With the updated image, I guess this example's FlexDirection is FlexDirectionType.Column.

Reply:
Yes, you are right. `FlexDirectionType.Column` is used by default in `FlexContainer `that's why I'm not assigning this property here. @mijin85cho please let me know if I should add it anyway.

Reply:
@pkalota No, you don't need to add it here. I'd like to suggest to add the description that `Column` is the default property at line 213 or line 203.

For example in case of line 203,

This property specifies the main axis direction along which flex items are placed. The default property is `Column`.

How about this update?